make/config/pebble/chart/templates/deployment.yaml

```diff
@@ -33,10 +33,12 @@ spec:
             mountPath: /config
             readOnly: true
           readinessProbe:
-            tcpSocket:
+            httpGet:
+              path: /dir
               port: 14000
+              scheme: HTTPS
             initialDelaySeconds: 1
-            periodSeconds: 1
+            periodSeconds: 2
```

Comment:
Please explain why this change would be valuable/what's the diff

Reply:
The main difference is that the pebble logs now don't show the following errors:
```
2023/05/10 17:28:40 http: TLS handshake error from 10.244.0.1:46208: EOF
2023/05/10 17:28:41 http: TLS handshake error from 10.244.0.1:55126: EOF
2023/05/10 17:28:42 http: TLS handshake error from 10.244.0.1:55142: EOF
2023/05/10 17:28:43 http: TLS handshake error from 10.244.0.1:55150: EOF
```

Instead, we see the following log messages now:
```
Pebble 2023/05/10 17:30:25 GET /dir -> calling handler()
Pebble 2023/05/10 17:30:27 GET /dir -> calling handler()
Pebble 2023/05/10 17:30:29 GET /dir -> calling handler()
Pebble 2023/05/10 17:30:31 GET /dir -> calling handler()
Pebble 2023/05/10 17:30:33 GET /dir -> calling handler()
```